### ISSUE TITLE
OCP's software package doesn't allow non-standard character

### ISSUE THREAD

**Issue by longdafeng** [oceanbase/oceanbase#775]
when I upload the package obproxy-3.2.0-1.el7.x86_64 (1).rpm and start to install obproxy, 

it will fail to install obproxy.

```
2022-02-07 18:18:11.814  INFO 57 --- [pool-subtask-executor-thread-54,7a8f5bcc9cff4500,90578e7d5539] c.a.ocp.core.task.util.OcpAgentUtils     : [OcpAgentUtils.runCmd] svrIp=172.30.62.217, port=62888, user=root, cmd=curl -s -k --user xxx:xxx -o /tmp/obproxy-3.2.0-1.el7.x86_64 (1).rpm http://172.30.62.231:8080/api/v2/files/sys-package/obproxy-3.2.0-1.el7.x86_64 (1).rpm

2022-02-07 18:18:11.836 ERROR 57 --- [pool-subtask-executor-thread-54,7a8f5bcc9cff4500,90578e7d5539] c.a.ocp.core.task.util.OcpAgentUtils     : call pos method failed, server IP: 172.30.62.217, port: 62888, args: root curl -s -k --user xxx:xxx -o /tmp/obproxy-3.2.0-1.el7.x86_64 (1).rpm http://172.30.62.231:8080/api/v2/files/sys-package/obproxy-3.2.0-1.el7.x86_64 (1).rpm, code: 2, result: /tmp/40fa07f2-87ff-11ec-8e86-00163e015de6: line 1: syntax error near unexpected token `('
/tmp/40fa07f2-87ff-11ec-8e86-00163e015de6: line 1: `curl -s -k --user xxx:xxx -o /tmp/obproxy-3.2.0-1.el7.x86_64 (1).rpm http://172.30.62.231:8080/api/v2/files/sys-package/obproxy-3.2.0-1.el7.x86_64 (1).rpm'

2022-02-07 18:18:11.838  INFO 57 --- [pool-subtask-executor-thread-54,7a8f5bcc9cff4500,90578e7d5539] c.a.o.c.m.j.model.SubtaskInstanceEntity  : Set state for subtask: 359294, current state: RUNNING, new state: FAILED

2022-02-07 18:18:11.840  WARN 57 --- [pool-subtask-executor-thread-54,7a8f5bcc9cff4500,90578e7d5539] c.a.ocp.core.job.runner.RunnerFactory    : Execute task failed, subtask=SubtaskInstanceEntity{id=359294, name=Install rpm by package and version, state=FAILED, operation=EXECUTE, className=com.alipay.ocp.service.task.business.host.InstallRpmByPackageAndVersionTask, seriesId=5, startTime=2022-02-07T18:18:11.796+08:00, endTime=2022-02-07T18:18:11.840+08:00}, failedMessage=RPC execution failed.server: 172.30.62.217 port: 62888 command: root curl -s -k --user xxx:xxx -o /tmp/obproxy-3.2.0-1.el7.x86_64 (1).rpm http://172.30.62.231:8080/api/v2/files/sys-package/obproxy-3.2.0-1.el7.x86_64 (1).rpm code: 2 result: /tmp/40fa07f2-87ff-11ec-8e86-00163e015de6: line 1: syntax error near unexpected token `('
/tmp/40fa07f2-87ff-11ec-8e86-00163e015de6: line 1: `curl -s -k --user xxx:xxx -o /tmp/obproxy-3.2.0-1.el7.x86_64 (1).rpm http://172.30.62.231:8080/api/v2/files/sys-package/obproxy-3.2.0-1.el7.x86_64 (1).rpm'

com.alipay.ocp.core.exception.UnexpectedException: [OCP UnexpectedException]: status=500 INTERNAL_SERVER_ERROR, errorCode=COMMON_RPC_EXECUTE_FAILED, args=172.30.62.217,62888,root curl -s -k --user xxx:xxx -o /tmp/obproxy-3.2.0-1.el7.x86_64 (1).rpm http://172.30.62.231:8080/api/v2/files/sys-package/obproxy-3.2.0-1.el7.x86_64 (1).rpm,2,/tmp/40fa07f2-87ff-11ec-8e86-00163e015de6: line 1: syntax error near unexpected token `('
/tmp/40fa07f2-87ff-11ec-8e86-00163e015de6: line 1: `curl -s -k --user xxx:xxx -o /tmp/obproxy-3.2.0-1.el7.x86_64 (1).rpm http://172.30.62.231:8080/api/v2/files/sys-package/obproxy-3.2.0-1.el7.x86_64 (1).rpm'
	at com.alipay.ocp.core.task.util.OcpAgentUtils.executePosMethod(OcpAgentUtils.java:272) ~[ocp-core-3.1.1-20210916.jar!/:3.1.1-20210916]
	at com.alipay.ocp.core.task.util.OcpAgentUtils.callPosMethod(OcpAgentUtils.java:135) ~[ocp-core-3.1.1-20210916.jar!/:3.1.1-20210916]
	at com.alipay.ocp.core.task.util.OcpAgentUtils.runCmd(OcpAgentUtils.java:57) ~[ocp-core-3.1.1-20210916.jar!/:3.1.1-20210916]
	at 
com.alipay.ocp.service.compute.command.RpcCommandExecutor.execute(RpcCommandExecutor.java:38) ~[ocp-service-3.1.1-20210916.jar!/:3.1.1-20210916]
	at com.alipay.ocp.service.compute.command.Commands.lambda$downloadFile$0(Commands.java:146) ~[ocp-service-3.1.1-20210916.jar!/:3.1.1-20210916]
	at com.alipay.ocp.common.pattern.Retry.executeUntilWithLimit(Retry.java:71) ~[ocp-common-3.1.1-20210916.jar!/:3.1.1-20210916]
	at com.alipay.ocp.common.pattern.Retry.executeUntilWithLimit(Retry.java:52) ~[ocp-common-3.1.1-20210916.jar!/:3.1.1-20210916]
	at com.alipay.ocp.service.compute.command.Commands.downloadFile(Commands.java:145) ~[ocp-service-3.1.1-20210916.jar!/:3.1.1-20210916]
	at com.alipay.ocp.service.compute.command.Commands.downloadRpm(Commands.java:82) ~[ocp-service-3.1.1-20210916.jar!/:3.1.1-20210916]
	at com.alipay.ocp.service.compute.command.Commands.downloadAndInstallRpm(Commands.java:106) ~[ocp-service-3.1.1-20210916.jar!/:3.1.1-20210916]
	at com.alipay.ocp.service.task.business.host.InstallRpmByPackageAndVersionTask.run(InstallRpmByPackageAndVersionTask.java:68) ~[ocp-service-3.1.1-20210916.jar!/:3.1.1-20210916]
	at com.alipay.ocp.core.metadb.job.model.SubtaskInstanceEntity.run(SubtaskInstanceEntity.java:216) ~[ocp-core-3.1.1-20210916.jar!/:3.1.1-20210916]
	at com.alipay.ocp.core.job.runner.JavaTaskRunner.doExecute(JavaTaskRunner.java:26) ~[ocp-core-3.1.1-20210916.jar!/:3.1.1-20210916]
	at com.alipay.ocp.core.job.runner.JavaTaskRunner.run(JavaTaskRunner.java:20) ~[ocp-core-3.1.1-20210916.jar!/:3.1.1-20210916]
	at com.alipay.ocp.core.job.runner.RunnerFactory.doRun(RunnerFactory.java:103) ~[ocp-core-3.1.1-20210916.jar!/:3.1.1-20210916]
	at com.alipay.ocp.core.job.runner.RunnerFactory.redirectOutputIfNotSysSchedule(RunnerFactory.java:147) ~[ocp-core-3.1.1-20210916.jar!/:3.1.1-20210916]
	at com.alipay.ocp.core.job.runner.RunnerFactory.run(RunnerFactory.java:92) ~[ocp-core-3.1.1-20210916.jar!/:3.1.1-20210916]
	at com.alipay.ocp.core.job.coordinator.worker.subtask.ReadySubtaskWorker.lambda$submitTask$2(ReadySubtaskW
orker.java:123) ~[ocp-core-3.1.1-20210916.jar!/:3.1.1-20210916]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[na:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_312]
```

**Comment by longdafeng** [comment]
Failed to install obproxy until I remove the package in the software center and upload a standard package. 

It still failed to install obproxy even when I upload a good package and select this pack during install obproxy. 

I must delete this package in software center 
![image](https://user-images.githubusercontent.com/374091/152771371-1873727f-d991-4aa1-b769-310082b43a2b.png)

**Comment by chris-sun-star** [comment]
add precheck when upload package, fixed in version 3.1.1-ce-bp1
